scripts/bert/dataset.py

```diff
@@ -169,19 +174,19 @@ def __call__(self, line):
         # the entire model is fine-tuned.
         tokens = []
         segment_ids = []
-        tokens.append('[CLS]')
+        tokens.append(self._tokenizer.vocab.cls_token)
         segment_ids.append(0)
         for token in tokens_a:
             tokens.append(token)
             segment_ids.append(0)
-        tokens.append('[SEP]')
+        tokens.append(self._tokenizer.vocab.sep_token)
         segment_ids.append(0)
 
         if tokens_b:
             for token in tokens_b:
                 tokens.append(token)
                 segment_ids.append(1)
-            tokens.append('[SEP]')
+            tokens.append(self._tokenizer.vocab.sep_token)
```

Comment:
Might be easier to read if you save a vocab variable vocab=self._tokenizer...

Reply:
Since there's ongoing work for the Transform API, I suggest we make incremental improvements there.